xml/ha_config_basics.xml

```diff
@@ -2789,13 +2789,21 @@ group g-vm1-and-services vm1 vm1-sshd \
    monitoring them and thus be oblivious to their status. You can even stop
    all &pace; services on a node, and all daemons and processes
    originally started as &pace;-managed cluster resources will continue
-   to run. If you attempt to start &pace; services on a node while the
+   to run.
+  </para>
+  <para>However, as &corosync; is also stopped when you stop the &pace; service,
+   this might lead to unexpected results for some resources. For example, DLM
+   depends on the membership and messaging services provided by &corosync;. If
+   &corosync; stops, the DLM resource will assume a split-brain scenario and
+   trigger a fencing operation. To avoid this, manually put resources like DLM
+   into unmanaged mode.</para>
```

Comment:
Admin should manually stop DLM or bring it back from maintenance mode before stopping pacemaker service.